mobile/src/main/res/values/do_not_translate.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <string name="twitter_token_format" formatted="false" translatable="false">oauth_token=%1$s,oauth_secret=%2$s</string>
     <string name="facebook_token_format" formatted="false" translatable="false">access_token=%s</string>
+    <bool name="setting_out_of_stock_warning_default" translatable="false">true</bool>
```

Comment:
I would inline this on the preferences. We'll never run into a situation where we'd want it true on a tablet but false on a phone or something like that, and there's no value to reusing this